src/components/pages/MissionPage.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line react-hooks/exhaustive-deps
+
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import { FetchMissions, selectMission } from '../../Redux/Mission/Mission';
+import Mission from '../Mission';
+
+const MissionPage = () => {
+  const missions = useSelector(selectMission, shallowEqual);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(FetchMissions());
+  }, [dispatch]);
```

Comment:
- [ ] Kindly move the use effect that fetches missions to the app component so that the fetch action dispatch once, as per `this` requirement.
![image](https://user-images.githubusercontent.com/24703171/174136630-83a46918-b145-4f1a-b501-cc5e2d17df08.png)

Reply:
@gemmen29 implemented the required changes thank you